### ISSUE TITLE
Overpass

### ISSUE THREAD

**Issue by DaveF63** [systemed/potlatch3#7]
I'm having problems with OP routines.
The 'bbox' appears to return a much larger area than the area zoomed into: 
https://www.openstreetmap.org/#map=19/51.37920/-2.38023

On running the routine P3's screen turns pale yellow & hangs for a few seconds where I'm unable to move the P3 window. It does display the 'task' box 
with the back & forward buttons & the way's ids and I can click through the 
results but the screen stays yellow.

Example:
[out:json][bbox:51.2892,-2.5241,51.4691,-2.236];
way[railway=platform];
out geom;

----
On selecting 'Edit with Remote Control, could P3's window be forced to the front?

DaveF[

**Comment by mmd-osm** [comment]
`out geom;` is an Overpass specific OSM XML format extension. I doubt Potlatch can parse this format.

**Comment by systemed** [comment]
Yep - P3 expects a lat/lon to be present in the Overpass result. That happens by default for nodes, `out center` will do it for ways, and as @mmd-osm says `out meta` does too.

The "Insert bbox" button inserts a bounding box around the current centre point, with the radius you specify in the adjacent text field. This defaults to 10km but you can adjust it to anything you like.

I'll write some docs for the Overpass integration in due course but have been concentrating on getting P3 out the door first. :)
